packages/components/RadioGroup/src/RadioButton.tsx

```diff
@@ -41,10 +41,13 @@ export const RadioButton = compose<IRadioButtonType>({
       }
     }, []);
 
+    // Ensure focus is placed on button after click
+    const changeSelectionWithFocus = useOnPressWithFocus(componentRef, changeSelection);
+
     /* RadioButton changes selection when focus is moved between each RadioButton and on a click */
     const pressable = useAsPressable({
       ...rest,
-      onPress: changeSelection,
+      onPress: changeSelectionWithFocus,
```

Comment:
I would double-check that the RadioButton gets focus when navigating with keyboard as well. When arrowing up/down in a RadioGroup, selection+focus should change

Reply:
It does